src/addrman.cpp

```diff
@@ -54,11 +54,8 @@ double CAddrInfo::GetChance(int64_t nNow) const
 {
     double fChance = 1.0;
 
-    int64_t nSinceLastSeen = nNow - nTime;
     int64_t nSinceLastTry = nNow - nLastTry;
```

Comment:
Maybe initializing `nSinceLastTry` and the following `if` statement can be collapsed into a call to `std::max()`.